src/components/ChatApp/MessageListItem.react.js

```diff
@@ -78,7 +78,9 @@ function drawTiles(tilesData){
                 )}
               <div className='tile-text'>
                 <p className='tile-title'>
-                  <strong>{processText(tile.title)}</strong>
+                  <a href={tile.link} target='_blank'
+                  rel='noopener noreferrer'><strong>{processText(tile.title)}</strong>
+                  </a>
                 </p>
                 {processText(tile.description)}<br/>
                 <a href={tile.link} target='_blank'
```

Comment:
In google assistant, the entire tile is hyperlinked, not the heading. So we must hyperlink the entire <Paper> or <div> i guess